Bencodex.Tests/Types/DictionaryTest.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using Bencodex.Types;
 using Xunit;
+using Boolean = Bencodex.Types.Boolean;
```

Comment:
Was this intended?  I guess you were intended to use qualified names instead, as you already did it in the below code!